tests/features/logIn.spec.ts

```diff
@@ -2,16 +2,36 @@ import { test, expect, Page } from '@playwright/test';
 import LoginPage from '../poms/LoginPage/LoginPage';
 import MainDashboard from '../poms/MainDashboard/MainDashboard';
 import { USER_NAME, USER_PWD } from '../data';
-
+import { pwdLabelTxt, userNameLabelTxt, wrongCredentialsError } from '../../constants';
 
 test.describe('Log in test suite', () => {
     test('Log in to the app', async ({page}) => {
         const loginPage = new LoginPage(page);
         const mainDashboard = new MainDashboard(page);
```

Comment:
The best solution would be to use fixtures for creation of the PO instances. If you do this you don't even need beforeEach.
https://www.youtube.com/watch?v=2O7dyz6XO2s